tests/tt_eager/python_api_testing/sweep_tests/test_configs/ci_sweep_tests_working/grayskull/pytorch_untilize_test.yaml

```diff
@@ -19,10 +19,36 @@ test-list:
           high: 100
       args-gen: gen_dtype_layout_device
       comparison:
-        function: comp_equal
+        function: comp_pcc
       output-file: untilize_sweep.csv
       args:
-        data-layout: ["ROW_MAJOR", "TILE"]
+        data-layout: ["TILE"]
+        data-type: ["BFLOAT16", "BFLOAT8_B"]
+        buffer-type: ["DRAM", "L1", "SYSTEM_MEMORY"]
+        out-buffer-type: ["DRAM"]
+  - untilize:
+      shape:
+        start-shape: [1, 1, 32, 32]
+        end-shape: [6, 12, 256, 256]
+        interval: [1, 1, 32, 32]
+        num-shapes: 1
+        num-samples: 128
+        args-sampling-strategy: "all"
+      env:
+        TT_METAL_SLOW_DISPATCH_MODE: ["", "1"]
+        # TT_PCI_DMA_BUF_SIZE: "1048576"
+      datagen:
+        function: gen_rand
+        dtype: bfloat16
+        args:
+          low: -100
+          high: 100
+      args-gen: gen_dtype_layout_device
+      comparison:
+        function: comp_pcc
```

Comment:
Would be good if for bfloat16 we could use comp_equal, or check pcc is 1 for these tms instead of using same comparison/threshold as bfp8

Reply:
Yes, sure, it makes sense I will update file